developer-model-service/docs/access.md

```diff
@@ -0,0 +1,139 @@
+# Access & Usage Information
+
+## Basic Information
+
+### Model Server URL
+
+The model server can be accessed [here](https://model-service.apps.domain.com/).
+
+## Authentication
+
+In order to gain access to the model server you will need to sign in with your SSO credentials by selecting Google auth. Once you have signed in you are able to generate a token by navigating to **`Apps and API Keys`** on the top ribbon and hitting **`Create new Application`**:
+
+![Generation Example](../images/generation-example.png)
+
+Once a token has been generated, you will be provided with the API server URL.
+
+**Note:** If you are unable to access the model server via SSO you need to reach out to a platform administrator.
+
+## API Schema
+<!--
+The name of the api, model-service-api, is grabbed from the name field in the ai-catalog.yaml metadata for the api.
```

Comment:
You deleted `ai-catalog.yaml` in this PR

this comment should be fixed

Reply:
Oops, yeah, that just needs to be renamed. I've updated it accordingly.